docs/contributors/getting-started.md

```diff
@@ -95,6 +95,14 @@ With the extension installed, ESLint will use the [.eslintrc.js](https://github.
     },
 ```
 
+**Note for Windows users:** In some environments, you'll find a lot of ESLint errors even after a fresh clone of the repository. In many cases, it's because of file line endings. Gutenberg uses `lf`, not `crlf` (the Windows default). In this case, you need to:
+
+1. Turn off the autocrlf option by executing `git config --global core.autocrlf false` on your command prompt.
```

Comment:
Not sure if this is a good advice.
1. It is global setting which means all cloned repositories will be affected. This doesn't matter if the contributor contributes only to Gutenberg, but that's rarely the case. If any of the other cloned repositories depend on Git for converting line breaks, chances are they will end up with problems.
2. Github advises against it: https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings. For Windows the advice is to use `git config --global core.autocrlf true`.
3. What happens to files where there is no linting or the linting doesn't check line endings? For example .scc/.scss, or .md files like this one? They may end up with mixed line endings if ever edited on Windows. Same for any auto-generated files that are committed.

Thinking that setting `git config --global core.autocrlf false` on Windows would potentially bring more problems than it solves. Seems better to let Git manage that on Windows (as it is now), and fix ESLint config to not interfere.

Reply:
Then, how about creating `.gitattributes` files as [Github suggests](https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings#per-repository-settings)?

```
# default behavior
* text eol=lf

# Denote all files that are truly binary and should not be modified.
*.png binary
*.jpg binary
```

Reply:
Yeah, this looks like a better option. It is per-repository (so we don't force the contributors to change their system's git settings), and takes away some of the text/binary guesswork. Will test on Win10 tomorrow. 

Would need to warn Windows users that line endings will change, and point them to the GitHub help. Also to make sure that nothing changes for Mac users :)